http/middleware/brotli.go

```diff
@@ -0,0 +1,43 @@
+package middleware
+
+import (
+	gbrotli "github.com/anargu/gin-brotli"
+	"github.com/andybalholm/brotli"
+	"github.com/gin-gonic/gin"
+
+	httpcontract "github.com/goravel/framework/contracts/http"
+	"github.com/goravel/framework/facades"
+	"github.com/goravel/framework/http"
+)
+
+func Brotli() httpcontract.Middleware {
+	return func(ctx httpcontract.Context) {
+		switch ctx := ctx.(type) {
+		case *http.GinContext:
+			newBrotli(gbrotli.Options{
+				WriterOptions: brotli.WriterOptions{
+					Quality: facades.Config().GetInt("brotli.compression_level"),
+					LGWin:   facades.Config().GetInt("brotli.lgwin_level"),
```

Comment:
S: We need to use `http.ConfigFacade` instead of `facades.Config()`.

Reply:
Just wondering why should we use http.ConfigFacade, is it about performance reasons ?

Reply:
`http.ConfigFacade` is implemented by the container, there is an import circle error if we use `facades.Config()` directly.